account-kit/smart-contracts/src/light-account/clients/multiOwnerLightAccount.ts

```diff
@@ -85,10 +132,22 @@ export async function createMultiOwnerLightAccountClient(
     chain,
   });
 
-  return createSmartAccountClient({
+  const account: SmartAccountClient = createSmartAccountClient({
```

Comment:
nit: naming this `account` is confusing since it's actually an SCA client

Reply:
also for consitency, can we do what we do with the single owner LA client method?
```
define light account instance

if transport is alchemy
  return alchemy client

return sca client
```

Reply:
Yup, doing it correct now, I just was placing it in a variable for the time, went back to the return. Also, keep an 👁️ out on any code that I do, I am bad at naming things like [so many others](https://martinfowler.com/bliki/TwoHardThings.html)